### ISSUE TITLE
helpers.parse_transaction_envelope_from_xdr with claimable balance

### ISSUE THREAD

**Issue by eugene3141** [StellarCN/py-stellar-base#464]
**Describe the bug**
helpers.parse_transaction_envelope_from_xdr with claimable balance returns
stellar_sdk.exceptions.ValueError: 180 is not a valid ClaimPredicateType, please upgrade the SDK or submit an issue here: https://github.com/StellarCN/py-stellar-base/issues.

**What version of dependencies are you using?**
stellar-sdk==3.2.0

**To Reproduce**
```
from stellar_sdk import Account, TransactionBuilder, Keypair, ClaimPredicate, Claimant, Asset, Network, helpers

sponsor_secret = "SAOJHTVFCYVKUMPNQI7RUSI566GKWXP7RXOHP4SV6JAVUQKSIWGPZFPJ"
claimant_secret = "SBOLGU7D7A7MTY4JZ3WZUKSKB6NZBQFNQG3BZT4HZW4AAVZJRG7TWXGQ"

sponsor_keypair = Keypair.from_secret(sponsor_secret)
claimant_keypair = Keypair.from_secret(claimant_secret)

network_passphrase = Network.TESTNET_NETWORK_PASSPHRASE

sponsor_account = Account(account_id=sponsor_keypair.public_key, sequence=1234567)

predicate_left = ClaimPredicate.predicate_before_relative_time(60 * 60 * 24 * 7)
predicate_right = ClaimPredicate.predicate_not(ClaimPredicate.predicate_before_relative_time(60 * 3))
predicate = ClaimPredicate.predicate_and(predicate_left, predicate_right)
claimant = Claimant(destination=claimant_keypair.public_key, predicate=predicate)
create_claimable_balance_te = TransactionBuilder(
    source_account=sponsor_account,
    network_passphrase=network_passphrase
).append_create_claimable_balance_op(
    asset=Asset.native(),
    amount="100",
    claimants=[claimant],
    source=sponsor_keypair.public_key
).build()
create_claimable_balance_te.sign(sponsor_keypair)

xdr = create_claimable_balance_te.to_xdr()

transactionEnvelop = helpers.parse_transaction_envelope_from_xdr(xdr, network_passphrase)
```

**Comment by overcat** [comment]
Thanks for your feedback, the v3 version uses the new XDR generator, which caused this bug, and I am already working on fixing it. You can use the v2 version first, thank you.

https://github.com/StellarCN/py-stellar-base/tree/v2

```
pip install stellar-sdk==2.12.0
```

**Comment by overcat** [comment]
I have found the root of the problem and will release an update in a few hours.

**Comment by overcat** [comment]
We fixed this issue in [3.2.1](https://github.com/StellarCN/py-stellar-base/releases/tag/3.2.1), and apologize for the inconvenience caused.

PyPi Package: https://pypi.org/project/stellar-sdk/3.2.1/

**Comment by eugene3141** [comment]
Thank you!
